lib/bencher_json/src/project/metric_kind.rs

```diff
@@ -55,6 +55,124 @@ static THROUGHPUT_UNITS: Lazy<NonEmpty> = Lazy::new(|| {
         .expect("Failed to parse metric kind units.")
 });
 
+pub const INSTRUCTIONS_NAME_STR: &str = "Instructions";
+#[allow(clippy::expect_used)]
+static INSTRUCTIONS_NAME: Lazy<NonEmpty> = Lazy::new(|| {
+    INSTRUCTIONS_NAME_STR
+        .parse()
+        .expect("Failed to parse metric kind name.")
+});
+pub const INSTRUCTIONS_SLUG_STR: &str = "instructions";
+#[allow(clippy::expect_used)]
+static INSTRUCTIONS_SLUG: Lazy<Option<Slug>> = Lazy::new(|| {
+    Some(
+        INSTRUCTIONS_SLUG_STR
+            .parse()
+            .expect("Failed to parse metric kind slug."),
+    )
+});
+pub const INSTRUCTIONS_UNITS_STR: &str = "instructions";
+#[allow(clippy::expect_used)]
+static INSTRUCTIONS_UNITS: Lazy<NonEmpty> = Lazy::new(|| {
+    INSTRUCTIONS_UNITS_STR
+        .parse()
+        .expect("Failed to parse metric kind units.")
+});
+pub const CYCLES_NAME_STR: &str = "Cycles";
+#[allow(clippy::expect_used)]
+static CYCLES_NAME: Lazy<NonEmpty> = Lazy::new(|| {
+    CYCLES_NAME_STR
+        .parse()
+        .expect("Failed to parse metric kind name.")
+});
+pub const CYCLES_SLUG_STR: &str = "cycles";
+#[allow(clippy::expect_used)]
+static CYCLES_SLUG: Lazy<Option<Slug>> = Lazy::new(|| {
+    Some(
+        CYCLES_SLUG_STR
+            .parse()
+            .expect("Failed to parse metric kind slug."),
+    )
+});
+pub const CYCLES_UNITS_STR: &str = "cycles";
+#[allow(clippy::expect_used)]
+static CYCLES_UNITS: Lazy<NonEmpty> = Lazy::new(|| {
+    CYCLES_UNITS_STR
+        .parse()
+        .expect("Failed to parse metric kind units.")
+});
+pub const L1_ACCESSES_NAME_STR: &str = "L1 Accesses";
+#[allow(clippy::expect_used)]
+static L1_ACCESSES_NAME: Lazy<NonEmpty> = Lazy::new(|| {
+    L1_ACCESSES_NAME_STR
+        .parse()
+        .expect("Failed to parse metric kind name.")
+});
+
+pub const ACCESSES_UNITS_STR: &str = "accesses";
+pub const L1_ACCESSES_SLUG_STR: &str = "l1-accesses";
+#[allow(clippy::expect_used)]
+static L1_ACCESSES_SLUG: Lazy<Option<Slug>> = Lazy::new(|| {
+    Some(
+        L1_ACCESSES_SLUG_STR
+            .parse()
+            .expect("Failed to parse metric kind slug."),
+    )
+});
+pub const L1_ACCESSES_UNITS_STR: &str = ACCESSES_UNITS_STR;
```

Comment:
I don't think these aliases are necessary, just using `ACCESSES_UNITS_STR` below should be fine.